README.md

```diff
@@ -40,3 +42,15 @@ Commits and merges into `master` will be deployed to the production server, curr
     ```
 1. Upload tar archive to webroot via cPanel: `Files > File Manager > /100days.envirodatagov.org`
 1. Extract archive within interface. (This will overwrite existing files.)
+
+### Scripted
+
+1. Set the shell environment variables in [`sample.env`](sample.env)
```

Comment:
This feels opaque. How do you set them if you don't already have access?

Reply:
Good call. I cleaned up a bit and am now using a rake task, which is a little more the ruby way